types/index.d.ts

```diff
@@ -528,6 +528,15 @@ declare module 'cloudinary' {
         [futureKey: string]: any;
     }
 
+    export interface AuthTokenApiOptions {
+        key: string;
+        acl: string;
+        ip?: string;
+        start_time?: number;
+        duration?: number;
+        expiration?: number;
```

Comment:
There's another optional property for AuthToken: url

Please add `url?: string` for completeness so we could merge it.

Reply:
Thanks for your review.
I've also added `url?: string;` .